src/app/shared/filestack/filestack.module.ts

```diff
@@ -2,6 +2,7 @@ import { IonicModule } from '@ionic/angular';
 import { ModuleWithProviders, NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { FilestackService } from './filestack.service';
+import { SharedModule } from '@shared/shared.module';
```

Comment:
When we import the shared module, we can remove the `common module, ionic module` those that is imported in the shared module already